torch/nn/quantized/functional.py

```diff
@@ -452,6 +452,25 @@ def hardswish(input, scale, zero_point):
         raise ValueError("Input to 'quantized.hardswish' must be quantized!")
     return torch._ops.ops.quantized.hardswish(input, scale, zero_point)
 
+def threshold(input, threshold, value):
+    # type: (Tensor, float, float) -> Tensor
+    r"""Applies the quantized version of the threshold function element-wise:
+
+    .. math::
+        \text{threshold}(x) = x \text{ if } x > \text{threshold else } \text{ value}
+
+    Args:
+        input: quantized input
+        threshold: the threshold value used to determine if :math:`x` should be
+        replaced by value
+        value: the value that is used in place of the original entries in x
```

Comment:
Instead of putting the details about the arguments, we can just say `See :class:`~torch.nn.Threshold` for more details.`. That way in case the documentation changes, it will be the same everywhere

Reply:
Got it! I've added in the change in the latest commit.